pom.xml

```diff
@@ -92,6 +92,11 @@
       <artifactId>netty-handler</artifactId>
       <version>${netty.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>${netty.version}</version>
+    </dependency>
```

Comment:
I think you need the classifier. Also, better make this dependency optional.

Reply:
I can make the dependency optional. But adding classifier based on ${os.name} will not build if someone is using non linux OS for development. I use OS X, it fails to find the dependency. Please let me know if there is a different way. I will address all other comments you have.